.github/ISSUE_TEMPLATE/1-addTokenForm.yml

```diff
@@ -9,11 +9,19 @@ body:
       value: |
         # Request to ADD token
         Thank you for contributing to CoW Swap's default token list.
-        
-        Please fill in all the required fields in the form. 
+
+        Please fill in all the required fields in the form.
         We will process and evaluate your request as soon as possible.
-        
-        Also, don't forget to update the issue title adding your token's `SYMBOL` and `NETWORK` (MAINNET or GNOSIS_CHAIN) 
+
+        Also, don't forget to update the issue title adding your token's `SYMBOL` and `NETWORK` (MAINNET or GNOSIS_CHAIN)
```

Comment:
maybe good to have a link to our source of truth for supported chains (if we have one), as this is the typical hard thing to maintain when new chains are available